resources/policy/bundles/asm-policy-v0.0.1/1.4.1_asm-request-authn-prohibited-output-headers.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: AsmRequestAuthnProhibitedOutputHeaders
+metadata:
+  name: asm-policy-v0.0.1-asm-request-authn-prohibited-output-headers
+  labels:
+    policycontroller.gke.io/bundleName: asm-policy-v0.0.1
```

Comment:
I do think we should potentially rename the scope of these annotations from `policycontroller.gke.io` to `policy.plural.sh`

Reply:
done